wazo_calld_client/commands/calls.py

```diff
@@ -220,3 +224,41 @@ def stop_record_from_user(self, call_id):
         r = self.session.put(url, headers=headers)
         if r.status_code != 204:
             self.raise_from_response(r)
+
+    def park(
+        self,
+        call_id: str,
+        parking_id: str,
+        preferred_slot: str | None = None,
+        timeout: int | None = None,
+    ) -> Mapping:
+        headers = self._get_headers()
+        url = self._client.url(self.resource, call_id, 'park')
+        body = {
+            'parking_id': parking_id,
+            'preferred_slot': preferred_slot,
+            'timeout': timeout,
+        }
+        r = self.session.put(url, headers=headers, json=body)
+        if r.status_code != 200:
+            self.raise_from_response(r)
+        return r.json()
+
+    def park_collocutor_from_user(
```

Comment:
Should the function also be renamed to remove the "collocutor" word?

Reply:
Yes, should have